bin/component-cli

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+DOWNLOAD_URL=$(curl -s https://api.github.com/repos/gardener/component-cli/releases/latest | jq -r ".assets[] | select(.name | test(\"${componentcli-linux-amd64.gz}\")) | .browser_download_url")
```

Comment:
hint: for readability, it is a good idea to wrap to multiple lines (this works also inside expressions), like so:

```
DOWNLOAD_URL=$(curl -s "https......." \
  | jq -r ".assets[]" \
  | ....
)
```